pages/agent/v3/help/_lock_release.md

```diff
@@ -14,7 +14,7 @@ script.
 
 ### Usage
 
-`buildkite-agent lock release [key]`
+`buildkite-agent lock release [key] [token]`
 
 ### Description
```

Comment:
A suggestion you don't have to address here, but I think the description should mention how `token` fits in.

Reply:
Done.